tools/pdf_util.py

```
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+"""Extract pdf structure in XML format"""
+import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)